omod/src/main/webapp/resources/scripts/patientqueue.js

```diff
@@ -0,0 +1,35 @@
+var patientqueue = patientqueue || {};
+
+patientqueue.completePatientQueueDialog = null;
+patientqueue.patientId = null;
+
+
+patientqueue.showCompletePatientQueueDialog = function (patientId) {
+    patientqueue.patientId = patientId;
+    if (patientqueue.completePatientQueueDialog == null) {
+        patientqueue.createCompletePatientQueueDialog();
+    }
+    if(patientQueueId!==""){
+        patientqueue.completePatientQueueDialog.show();
+    }else {
```

Comment:
Why do we even show the link for a patient without an active queue?

Reply:
Currently the inclusion of data elements to be used in the evaluation of the show if logic that shows and hides links is not clear. As we investigate into that. The current solution is to throw a toast when a user tries to clink on the link